Project doesn't run on fedora linux due to javafx libraries

Running the project on intellij fedora linux throws the following error (possibly due to wayland)

```
02:32:13.796 [ERROR] [system.err] 
02:32:13.796 [ERROR] [system.err] (java:62295): Gdk-CRITICAL **: 02:32:13.796: gdk_x11_display_set_window_scale: assertion 'GDK_IS_X11_DISPLAY (display)' failed
```

--- a/build.gradle
+++ b/build.gradle
@@ -42,7 +42,7 @@ task coverage(type: JacocoReport) {
 
 dependencies {
     String jUnitVersion = '5.4.0'
-    String javaFxVersion = '11'
+    String javaFxVersion = '16'
 
     implementation group: 'org.openjfx', name: 'javafx-base', version: javaFxVersion, classifier: 'win'
     implementation group: 'org.openjfx', name: 'javafx-base', version: javaFxVersion, classifier: 'mac'

--- a/build.gradle
+++ b/build.gradle
@@ -2,7 +2,6 @@ plugins {
     id 'java'
     id 'checkstyle'
     id 'com.github.johnrengelman.shadow' version '4.0.4'
-    id 'org.openjfx.javafxplugin' version '0.0.9'
     id 'application'
     id 'jacoco'
 }
@@ -15,11 +14,6 @@ targetCompatibility = JavaVersion.VERSION_11
 repositories {
     mavenCentral()
     maven { url 'https://oss.sonatype.org/content/repositories/snapshots/' }
-}
-
-javafx {
-    version = "16"
-    modules = [ 'javafx.base', 'javafx.controls', 'javafx.fxml' ]
 }
 
 checkstyle {

--- a/build.gradle
+++ b/build.gradle
@@ -2,6 +2,7 @@ plugins {
     id 'java'
     id 'checkstyle'
     id 'com.github.johnrengelman.shadow' version '4.0.4'
+    id 'org.openjfx.javafxplugin' version '0.0.9'
     id 'application'
     id 'jacoco'
 }
@@ -14,6 +15,11 @@ targetCompatibility = JavaVersion.VERSION_11
 repositories {
     mavenCentral()
     maven { url 'https://oss.sonatype.org/content/repositories/snapshots/' }
+}
+
+javafx {
+    version = "16"
+    modules = [ 'javafx.base', 'javafx.controls', 'javafx.fxml' ]
 }
 
 checkstyle {

